must-gather/collection-scripts/gather

```diff
@@ -24,12 +26,14 @@ resources+=(objectbuckets)
 # Add general resources to list if necessary 
 
 # Run the Collection of Resources using must-gather
-for resource in ${resources[@]}; do
+for resource in "${resources[@]}"; do
     oc adm --dest-dir=${BASE_COLLECTION_PATH} inspect ${resource} --all-namespaces
 done
 
 # Call other gather scripts
 gather_noobaa_resources ${BASE_COLLECTION_PATH}
 gather_ceph_resources ${BASE_COLLECTION_PATH}
 
+find ${BASE_COLLECTION_PATH} -type f -empty -delete
+find ${BASE_COLLECTION_PATH} -type d -empty -delete
```

Comment:
This is actually the only hunk of this patch that relates to the commit or even PR. This should be a separate commit, and probably the only commit in this PR.

Reply:
One possible small improvement: the linux manpage of find says this about `-empty`:
```
-empty File is empty and is either a regular file or a directory.
```
So it should be enough to just have
```
find ${BASE_COLLECTION_PATH} -empty -delete
```

Also please enclose the `${BASE_COLLECTION_PATH}` in double quotes (`"`). The script is not shellcheck proof, but we should at least not add more cases that would need to be fixed later.

Reply:
yes removing the `-type` and keeping only a single statement of the find will do the work. But I think it will miss deleting the directories which contain only a single empty file right ?

Reply:
i think i m wrong! tested it, it is working as expected. Updated the PR.